packages/react-components/src/organisms/UserProductivityTable.tsx

```diff
@@ -0,0 +1,164 @@
+import { css } from '@emotion/react';
+import { Card } from '../atoms';
+import { charcoal, lead, neutral200, steel } from '../colors';
+import { rem, tabletScreen } from '../pixels';
+import { borderRadius } from '../card';
+import { alumniBadgeIcon, inactiveBadgeIcon } from '../icons';
+
+const container = css({
+  display: 'grid',
+  paddingTop: rem(32),
+});
+
+const gridTitleStyles = css({
+  display: 'none',
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    display: 'inherit',
+    paddingBottom: rem(16),
+  },
+});
+
+const rowTitleStyles = css({
+  paddingTop: rem(32),
+  paddingBottom: rem(16),
+  ':first-of-type': { paddingTop: 0 },
+  [`@media (min-width: ${tabletScreen.min}px)`]: { display: 'none' },
+});
+
+const rowStyles = css({
+  display: 'grid',
+  padding: `${rem(20)} ${rem(24)} 0`,
+  borderBottom: `1px solid ${steel.rgb}`,
+  ':first-of-type': {
+    borderBottom: 'none',
+  },
+  ':nth-of-type(2n+3)': {
+    background: neutral200.rgb,
+  },
+  ':last-child': {
+    borderBottom: 'none',
+    marginBottom: 0,
+    paddingBottom: rem(15),
+    borderRadius: rem(borderRadius),
+  },
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    gridTemplateColumns: '1fr 1fr 1fr 1fr 1fr 1fr',
```

Comment:
I wonder if all the columns should have the same width. On figma `ASAP Public Output` is displayed in one line and it this column seems to be bigger than the others. While `Ratio` seems to be smaller.

<img width="899" alt="Screenshot 2024-04-03 at 11 53 52" src="https://github.com/yldio/asap-hub/assets/16595804/c935239f-4c42-4345-9233-1b7542e75894">

Here on the hub, `ASAP Public Output` doesn't fit in a single line in a 13inch screen
<img width="1048" alt="Screenshot 2024-04-03 at 11 54 40" src="https://github.com/yldio/asap-hub/assets/16595804/b2e5faf2-9718-4e6d-b622-ae657665229c">